src/main/app/package.json

```diff
@@ -3,11 +3,13 @@
   "version": "0.0.1",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve --proxy-config proxy.conf.json",
-    "build": "ng build --configuration=production --base-href=/",
+    "dev": "npm run generate:types:zac-openapi && ng serve --hmr --proxy-config proxy.conf.json",
+    "start": "npm run generate:types:zac-openapi && ng serve --proxy-config proxy.conf.json",
+    "build": "npm run generate:types:zac-openapi && ng build --configuration=production --base-href=/",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e"
+    "e2e": "ng e2e",
+    "generate:types:zac-openapi": "npx openapi-typescript ./../../../build/generated/openapi/META-INF/openapi/openapi.yaml -o ./src/generated/types/zac-openapi-types.d.ts"
```

Comment:
The `./` 's here mean that you can only run this command from the root project folder right? I guess that could be ok (more or less) but would be good I think to add a little comment above this line stating that fact?

Reply:
there are no comments in json unfortunately... this commands needs to be run from the angular folder, which is documented, you will never be able to run this outside of this folder, npm will not recognise this command

Reply:
`.jsonc` when